launcher/rockcraft.yaml

```
@@ -0,0 +1,73 @@
+# Based on: https://github.com/kubeflow/pipelines/blob/2.2.0/backend/Dockerfile.launcher
+name: kfp-launcher
+summary: Kubeflow Pipelines Launcher
+description: |
+  Kubeflow Pipelines Launcher is a streamlined interface designed to simplify the creation, management, and deployment of machine learning workflows on Kubernetes.
+version: v2.2.0
```

I missed this one in my last review, other than that LGTM!

```suggestion
version: 2.2.0
```